### PR TITLE
austin.intro.2.Fiero.Steven

### PR DESCRIPTION
**Pull request by ssfiero**

### CODE REVIEW

File: 02LessonTwo/homework/index.html [hunk 1]
```diff
@@ -1,10 +1,60 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Steven's Webpage</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
+
+<body background="https://images.unsplash.com/photo-1422228527100-2ecacc4c92a3?dpr=1&auto=format&crop=entropy&fit=crop&w=1500&h=1120&q=80"/>
```
Comment (fasteddiegarcia) [inline 02LessonTwo/homework/index.html:8]: do this in CSS in the body { }

File: 02LessonTwo/homework/index.html [hunk 2]
```diff
@@ -1,10 +1,60 @@
...
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><a href="http://www.aclfestival.com/"/><img src="http://aclp3.c3cdn.com/wp/wp-content/uploads/2016/02/ACL16-Facebook-Share-Image-1200x629v2.jpg"/></td>
+          <td><a href="http://www.wired.com"/><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/95/Wired_logo.svg/2000px-Wired_logo.svg.png"/></td>
+          <td><a href="http://www.austincodingacademy.com/"/><img src="https://i.ytimg.com/vi/29wiPl5vKos/maxresdefault.jpg"/></td>
+          <td><a href="http://austinartgarage.com/"/><img src="https://pbs.twimg.com/profile_images/3603562132/a370e03a7f8b9003deffbc176fd8da81_400x400.jpeg"/></td>
+          <td><a href="http://www.txstate.edu"/><img src="http://d54ha93qrghds.cloudfront.net/variant/large/otter-samsung-galaxy-s6-com/black/texas-state-design1_color_3.jpg"/></td>
+        </tr>
```
Comment (fasteddiegarcia) [inline 02LessonTwo/homework/index.html:48]: fantastic use of table and background image. everywhere i looked there was something to see.

File: 03LessonThree/in-class-project/daily-planet.html [hunk 3]
```diff
@@ -1,10 +1,45 @@
...
+    <p style="font-size:10px">EWEN ROBERTS/FLICKR UNDER A CC BY 2.0 LICENSE</p>
+    <a href="https://www.facebook.com/"><img src="https://sunypoly.edu/home5/cmsfiles/favicon.ico/fb-logo.png" alt="facebook link" height ="20px" width="20px"/></a>
+    <a href="https://twitter.com/?lang=en"><img src="https://upload.wikimedia.org/wikipedia/en/thumb/9/9f/Twitter_bird_logo_2012.svg/1259px-Twitter_bird_logo_2012.svg.png" alt="twitter link" height ="20px" width="20px"/></a>
+    <a href="https://www.instagram.com/accounts/login/"><img src="https://lh3.googleusercontent.com/aYbdIM1abwyVSUZLDKoE0CDZGRhlkpsaPOg9tNnBktUQYsXflwknnOn2Ge1Yr7rImGk=w300" alt="instagram link" height ="20px" width="20px"/></a>
+    <a href="https://www.linkedin.com/"><img src="http://theappshowbox.com/wp-content/uploads/2015/10/LinkedIn-Logo.png" alt="Linkedin link" height ="20px" width="20px"/></a>
+    <p>© 2013-2016 AOL Inc. All rights reserved. Aol Tech Privacy Policy About Our Ads Anti Harassment Policy Terms of Service Powered by WordPress.com VIP Fonts by Superman!</p>
+    </footer>
+</body>
+
+</html>
```
Comment (ethanstenis) [inline 03LessonThree/in-class-project/daily-planet.html:45]: Good use of h6 tags to emulate bylines. The animated gif is also a nice touch! Great work.

File: 04LessonFour/in-class-project/css/style.css [hunk 4]
```diff
@@ -1,84 +1,37 @@
...
+
+.support {
+  background-color: #D3D3D3;
+  margin-top: -6px;
+  padding-left: 10px;
+}
+
+.support a {
+  color: blue;
+}
```
Comment (ethanstenis) [inline 04LessonFour/in-class-project/css/style.css:31]: Add some padding to your support class to give the text some space.

**Pull request closed without merging**